src/net.h

```diff
@@ -1002,18 +1002,31 @@ class CNode
 };
 
 /**
- * Interface for message handling
+ * Interface for message handling.
+ * For a given `CNode`, the methods must be called in the following order:
```

Comment:
I think this would be better phrased as requirements for implementers of the interface, rather than the user of the interface. ie: "For a given `CNode`, the methods will be called in the following order:".

Reply:
Changed.